app/src/main/res/drawable/ic_language.xml

```diff
@@ -0,0 +1,3 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
+    <path android:pathData="M2.375 12c0-5.316 4.31-9.625 9.625-9.625 5.316 0 9.625 4.31 9.625 9.625 0 5.316-4.31 9.625-9.625 9.625-5.316 0-9.625-4.31-9.625-9.625zm1.273-0.625h2.903c0.041-0.94 0.175-1.854 0.392-2.733-0.687-0.363-1.33-0.793-1.918-1.28-0.776 1.166-1.268 2.536-1.377 4.013zm0 1.25h2.903c0.04 0.88 0.161 1.739 0.357 2.568-0.706 0.364-1.368 0.803-1.974 1.305-0.723-1.134-1.181-2.455-1.286-3.873zm4.155 0c0.034 0.7 0.125 1.384 0.27 2.048 1.041-0.394 2.154-0.635 3.302-0.696v-1.352H7.803zM5.684 17.5c0.486-0.408 1.015-0.77 1.576-1.079 0.41 1.212 0.982 2.347 1.69 3.382-1.268-0.497-2.386-1.293-3.266-2.303zM8.4 15.887c0.536 1.648 1.403 3.145 2.522 4.42 0.15 0.019 0.3 0.034 0.453 0.045v-5.123c-1.036 0.062-2.038 0.29-2.975 0.658zM8.102 9.172c-0.162 0.712-0.263 1.448-0.3 2.203h3.573V9.901c-1.151-0.077-2.25-0.33-3.273-0.729zm3.273-0.525C10.342 8.57 9.357 8.332 8.442 7.962c0.532-1.58 1.372-3.019 2.457-4.265 0.157-0.02 0.316-0.037 0.476-0.049v5zm-5.581-2.27C6.65 5.43 7.722 4.682 8.93 4.206c-0.675 0.99-1.222 2.067-1.622 3.212-0.54-0.302-1.046-0.652-1.514-1.041zm11.71 6.248h2.848c-0.106 1.434-0.572 2.767-1.31 3.91-0.585-0.486-1.225-0.916-1.91-1.28 0.206-0.847 0.333-1.727 0.373-2.63zm0.001-1.25h2.847c-0.106-1.437-0.574-2.772-1.314-3.916-0.59 0.478-1.23 0.897-1.912 1.247 0.21 0.86 0.339 1.752 0.379 2.669zm-1.542-2.153c0.156 0.697 0.255 1.416 0.29 2.153h-3.628V9.913c1.16-0.055 2.285-0.295 3.338-0.69zm0.804-1.74c0.538-0.293 1.046-0.635 1.515-1.02-0.854-0.969-1.93-1.738-3.146-2.23 0.679 1 1.23 2.09 1.63 3.25zm-3.615-3.778c1.093 1.251 1.94 2.708 2.475 4.309-0.946 0.367-1.958 0.593-3.002 0.648V3.648c0.177 0.013 0.353 0.032 0.527 0.056zm3.101 8.921c-0.035 0.717-0.13 1.418-0.28 2.098-1.044-0.409-2.17-0.667-3.348-0.74v-1.358h3.628zm-0.613 3.311c-0.939-0.382-1.952-0.626-3.015-0.7v5.116c0.174-0.013 0.346-0.03 0.517-0.054 1.104-1.26 1.962-2.737 2.498-4.362zm-0.514 3.836c1.224-0.493 2.304-1.266 3.162-2.24-0.468-0.392-0.974-0.744-1.514-1.048-0.403 1.176-0.96 2.28-1.648 3.288z" android:fillColor="#FAFAFA" android:fillType="evenOdd"/>
```

Comment:
Should this `VectorDrawable` get its color from a theme attribute, instead of a hardcoded `#FAFAFA`?